src/libraries/BotBuilder/Microsoft.Agents.BotBuilder/UserAuth/UserAuthorizationModuleLoader.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Agents.Authentication;
+using Microsoft.Agents.BotBuilder.Errors;
+using Microsoft.Agents.BotBuilder.UserAuth.TokenService;
+using Microsoft.Agents.Core.Errors;
+using Microsoft.Agents.Storage;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.Loader;
+
+namespace Microsoft.Agents.BotBuilder.UserAuth
+{
+    internal class UserAuthorizationModuleLoader( AssemblyLoadContext loadContext, ILogger logger)
```

Comment:
is there a situation where we would load a module from someone else?

Reply:
Yes.  For a different IUserAthorization impl.  Like TeamsSso.

Reply:
Lets leave alone until we make a decision on if we around going to allow other IUserAuthorization implementation.  Either from us or 3P.